.gitignore

```diff
@@ -18,4 +18,5 @@ package-lock.json
 .idea/*
 .idea/
 *.iml
-.DS_Store
\ No newline at end of file
+.DS_Store
+user/raidboss-example.js
```

Comment:
Can you remove this from here? The example is part of the repository, so I don't want to ignore changes to it.